source/release-notes-pxc-v8.0.23.upd.rst

```diff
@@ -0,0 +1,13 @@
+.. _pdpxc.release_notes_8.0.23:
+
+********************************************************************************
+|pdmysql| 8.0.23 using |PXC| Update
+********************************************************************************
+
+:Date: September 15, 2021
+:Installation: :ref:`installation`
+
+This update of the |PXC|-based variant of the |pdmysql| includes the HAProxy version 2.3.14, which fixes the security vulnerability `CVE-2021-40346 <https://security-tracker.debian.org/tracker/CVE-2021-40346>`_.
```

Comment:
```suggestion
This update of the |PXC|-based variant of the |pdmysql| includes the HAProxy version 2.3.14, which fixes a security vulnerability -see `CVE-2021-40346 <https://security-tracker.debian.org/tracker/CVE-2021-40346>`_ for details.
```

Reply:
But it fixes a specific CVE, so it should be "the" here.

Reply:
Good point, works for me :)